src/client/thirdParty/Inkeep.tsx

```diff
@@ -31,12 +33,23 @@ const useInkeepSettings = (): InkeepSharedSettings => {
   const integrationId = env.NEXT_PUBLIC_INKEEP_INTEGRATION_ID;
   const organizationId = env.NEXT_PUBLIC_INKEEP_ORG_ID;
 
+  const { data } = useVisitorData({ extendedResult: true, timeout: FPJS_CLIENT_TIMEOUT });
+  const visitorId = data?.visitorId || '';
+
+  const logEventCallback = (event: any) => {
+    const { name } = event.properties;
```

Comment:
Don't you need this check you had here previously? 

`if (event.eventName === GET_HELP_OPTIONS_CLICKED)`